spec/install/conf.ts

```diff
@@ -0,0 +1,7 @@
+import {Config} from 'protractor';
+
+export let config: Config = {
+  seleniumAddress: 'http://127.0.0.1:4444/wd/hub',
```

Comment:
Do we need to use a real address here? Can we either a) get it from the environment file,or b) just use the mock since we're not actually using the webdriver server?